ui/components/src/Drawer/Drawer.tsx

```diff
@@ -34,7 +34,6 @@ export const Drawer = ({ anchor = 'right', isOpen, onClose, PaperProps, children
         sx: combineSx(
           {
             width: `${DRAWER_DEFAULT_WIDTH}px`,
-            padding: (theme) => theme.spacing(2),
```

Comment:
This top level padding was causing issues with:
- positioning of the scrollbar for the content
- the border for the header was not fully filling the drawer